desktop/window-handlers/get-logs/index.js

```diff
@@ -0,0 +1,79 @@
+'use strict';
+
+/**
+ * External dependencies
+ */
+const path = require( 'path' );
+const { app, dialog } = require( 'electron' );
+
+/**
+ * Internal dependencies
+ */
+const state = require( 'lib/state' );
+const system = require( 'lib/system' );
+const { zipContents } = require( 'lib/archiver' );
+const log = require( 'lib/logger' )( 'desktop:get-logs' );
+
+/**
+ * Module variables
+ */
+const logPath = state.getLogPath();
+
+function pad( n ) {
+	return n < 10 ? '0' + n : n;
+}
```

Comment:
feel free to use the built-in method to clarify the intent

```js
const pad = n => `${n}`.padStart(2, '0');
```

Reply:
TIL 🤦‍♂

Updated in 4b23342